packages/functional-tests/pages/signupReact.ts

```diff
@@ -83,7 +83,7 @@ export class SignupReactPage extends BaseLayout {
     await this.submitButton.click();
   }
 
-  async fillOutSignupForm(password: string, age: string) {
+  async fillOutSignupForm(password: string, age = '21') {
```

Comment:
Not objecting, but I'll share that not setting the default to `21` was intentional on my part.
The theory being that inputs should be visible at the top level of a test for easier manual validation. In other words, top level reads like a set of instructions or test case.